package/rules/less.js

```diff
@@ -13,7 +13,11 @@ module.exports = canProcess('less-loader', (resolvedPath) =>
       loader: resolvedPath,
       options: {
         lessOptions: {
-          paths: [path.resolve(__dirname, 'node_modules'), sourcePath, ...paths]
+          paths: [
+            path.resolve(__dirname, 'node_modules'),
+            sourcePath,
+            ...paths
+          ]
```

Comment:
nit: could you revert this?

```suggestion
          paths: [path.resolve(__dirname, 'node_modules'), sourcePath, ...paths]
```